pkg/file/metadata_test.go

```diff
@@ -1,11 +1,14 @@
+// +build linux
```

Comment:
Should these also build mac?

Reply:
That's my fault - I'll update the tags to exclude windows rather than include all builds we want